src/main/java/org/spongepowered/api/event/network/ClientConnectionEvent.java

```diff
@@ -160,7 +179,23 @@
          * @param world The world
          * @param transform The transform
          */
-        void setLoginPosition(World world, Transform transform);
+        void setLocation(World world, Transform transform);
```

Comment:
Per my comment below, call this setLoginPosition. Or come up with a better combined method name. If you do, correct the other events.